src/IssueViz/Editor/FilteringTaggerBase.cs

```diff
@@ -120,6 +120,17 @@ private void OnAggregatorTagsChanged(object sender, BatchedTagsChangedEventArgs
 
         public IEnumerable<ITagSpan<TTagType>> GetTags(NormalizedSnapshotSpanCollection spans)
         {
+            if (disposedValue)
+            {
+                // See https://github.com/SonarSource/sonarlint-visualstudio/issues/1693
+                // Intermittently GetTags gets called even after the tagger instance has been
+                // disposed. We don't want an exception to be thrown in this case as it causes
+                // a gold bar to appear. To the user, it would look like the bug is in our code,
+                // whereas in fact VS has disposed the tagger but is continuing to call it.
+                Debug.Fail($"GetTags called on disposed tagger. Tagger type: {this.GetType().FullName}. File: { GetSnapshot().TextBuffer.GetFilePath()}");
```

Comment:
I think the Debug.Fail will be noisy. We can't do anything with it, that's VS's behavior. I think we should just do `yield break`.

Reply:
Yep, it will be noisy when it happens. I was undecided between `Debug.Fail` and `Debug.WriteLine` and decided to start with `Fail` because I want to assess how often this actually occurs and what the impact is.
I'll change the committed version to `WriteLine` and keep the `Fail` on my machine for now.

Reply:
But why do you need the debug at all? What will you do with the information?

Reply:
I'm interested in two things: 1) does it happen in real world usage, and 2) if so does it have a noticeable impact on the user experience.